### PR TITLE
fix(install/linux): properly parse `--root` flag and tag option

### PR DESCRIPTION
**Pull request by rxri**

### CODE REVIEW

**Review by afonsojramos — CHANGES_REQUESTED** [review]
What is this flag for? We have no documentation for it anywhere.

**Comment by rxri** [comment]
> What is this flag for? We have no documentation for it anywhere.

@afonsojramos how is it not... 
image

**Comment by afonsojramos** [comment]
Well, fair. I just meant that we don't show it in any `--help` command, and that we don't have it in the docs. But I suppose that for this case that is sort of the goal, as it is an exception to the rule.

**Comment by rxri** [comment]
Well yeah, it's intended tbh.

File: install.sh [hunk 1]
```diff
@@ -5,13 +5,23 @@
 
 set -e
 
+for arg in "$@"; do
+    shift
+    case "$arg" in
+        "--root") set -- "$@" "-r" ;;
+        *) set -- "$@" "$arg"
+    esac
+done
```
Comment (afonsojramos) [inline install.sh:14]: So all we're doing is to change the --root flag into -r?

File: install.sh [hunk 2]
```diff
@@ -5,13 +5,23 @@
...
+
 while getopts ":r" arg; do
     case "${arg}" in
         "r") override_root=1 ;;
```
Comment (afonsojramos) [inline install.sh:18]: Couldn't we just change this to `root`?

File: install.sh (hunk 1 above)
Comment (rxri) [reply install.sh:14]: yeah

File: install.sh (hunk 2 above)
Comment (rxri) [reply install.sh:18]: well, most of scripts/cli provide short version an long version so that's why it has both 🤷‍♀️

Comment (afonsojramos) [reply install.sh:18]: Yeah, but that doesn't always happens as it is a rule that doesn't apply to all flags. In this case, we are talking about a flag that shouldn't be used outside of exceptions, so I would say that we shouldn't have a short version for this one, and just simplify and only accept `--root`

Comment (rxri) [reply install.sh:18]: alright

**Review by afonsojramos — APPROVED** [review]

**Review by theRealPadster — APPROVED** [review]

**Pull request merged**